src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/ScopeController.cs

```diff
@@ -122,6 +122,11 @@ public async Task<IActionResult> Create(CreateOpenIdScopeViewModel model, string
                 ModelState.AddModelError(nameof(model.Name), S["The name is already taken by another scope."]);
             }
 
+            if(model.Name.Contains(' '))
```

Comment:
Consider moving that to the view model class using `IValidatableObject`.

Reply:
Changed the validation using an IValidatableObject